src/components/AddAsset/AddAssetDetailsCard.vue

```diff
@@ -120,5 +156,20 @@ export default class AddAssetDetailsCard extends Mixins(TranslationMixin, Loadin
   &_text {
     color: var(--s-color-status-warning);
   }
+
+  .asset-list-container {
+    & > div {
+      margin-bottom: 14px;
+    }
+    & > div:first-child {
+      margin-top: 14px;
+    }
+    & > div:last-child {
+      margin-bottom: 14px;
+    }
+  }
+  .asset-list-scrollbar {
+    height: 400px;
```

Comment:
`calc(var(--s-asset-item-height--fiat)* 2 + 32px);`
To save some space & show 2 assets + part of the third